extmod/modure.c

```diff
@@ -40,6 +40,7 @@
 #include "re1.5/re1.5.h"
 
 #define FLAG_DEBUG 0x1000
+#define FLAG_IGNORECASE 0x0100
```

Comment:
@dmazzella it looks like IGNORECASE is not used.  Do you use it in any of your scripts?

Reply:
not implemented, i will remove it